composer.json

```
@@ -16,7 +16,7 @@
     ],
     "require": {
         "php": ">=7.1",
-        "psr/http-message": "^1.0",
+        "psr/http-message": "^1.0 || ^2.0",
```

Do we need to write 1.1?

```suggestion
        "psr/http-message": "^1.1 || ^2.0",
```